garrysmod/gamemodes/sandbox/entities/entities/gmod_winch_controller.lua

```diff
@@ -1,7 +1,8 @@
 
 if ( CLIENT ) then return end
 
-ENT.Type = "point"
+ENT.PrintName 	= "Winch Controller"

```

Comment:
`PrintName` is clientside so this won't work here because of the `if` statement above.